include_all_in_one/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# This script will create the amalgamated library into 
+#    include_all_in_one/include/fplus/fplus.hpp
+#
+# Notes:
+# - include_all_in_one/include/fplus/fplus.hpp is part of the source repository, so that using FunctionalPlus 
+#   online with tools such as Compiler Explorer is easy. Just write:
+#     #include "https://raw.githubusercontent.com/Dobiasd/FunctionalPlus/master/include_all_in_one/include/fplus/fplus.hpp"
+# - this file will be updated automatically at each build, so that library developpers do not forget 
+#   to commit it (ideally once per PR)
+find_package (Python COMPONENTS Interpreter QUIET)
+if (Python_FOUND)
```

Comment:
Please format the commands in a way that is consistent with the rest of the project:

* No space after the command's name
* 2 spaces to indent
* 4 spaces for continuation